### PR TITLE
Optimize the for range code in generic_scheduler.go

### PR DESCRIPTION
**Pull request by jyz0309**
Signed-off-by: jyz0309 <45495947@qq.com>

#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

**Comment by jyz0309** [comment]
@chendave @ahg-g PTAL~

File: pkg/scheduler/generic_scheduler.go [hunk 1]
```diff
@@ -407,9 +407,9 @@ func prioritizeNodes(
 	// This is required to generate the priority list in the required format
 	if len(extenders) == 0 && !fwk.HasScorePlugins() {
 		result := make(framework.NodeScoreList, 0, len(nodes))
-		for i := range nodes {
+		for _, node := range nodes {
 			result = append(result, framework.NodeScore{
-				Name:  nodes[i].Name,
+				Name:  node.Name,
```
Comment (chendave) [inline pkg/scheduler/generic_scheduler.go:412]: looks like the different way to iterate a slice but I cannot see where is the optimization comes from.

Comment (jyz0309) [reply pkg/scheduler/generic_scheduler.go:412]: It’s just that I think it’s cleaner to iterate slice the code like this : ) If not, I will close this pr.🙈

**Pull request closed without merging**